src/js/components/Form/Form.js

```diff
@@ -460,9 +460,19 @@ const Form = forwardRef(
             componentValue !== undefined && // input driving
             componentValue !== formValue // don't already have it
           ) {
-            setValueState((prevValue) =>
-              setFieldValue(name, componentValue, prevValue),
-            );
+            // Check if componentValue & formValue are arrays
+            if (Array.isArray(componentValue) && Array.isArray(formValue)) {
```

Comment:
It feels like the string comparison should happen as a condition to enter the block at all. Maybe altering line 461 to something like:

 ```
((Array.isArray(componentValue) && Array.isArray(formValue) && componentValue.toString() !== formValue.toString()) || componentValue !== formValue)
```
then leaving everything inside the block as it was before this PR

Reply:
makes sense and simplifies I'll make adjustments